pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -761,36 +761,41 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
      * Emits the TokenAdded event.
      *
      * @param token - The ERC20 token to be added to the Pool.
+     * @param assetManager - The asset manager for the token.
```

Comment:
We should add some natspec to here explaining that they need an asset manager which is able to add  the neccessary tokens (and reduce the token balance to zero again if they want to remove the token in future). nbd if it's in a separate PR later.